### DISCUSSION TITLE
Installer detect CR-10 SE as KE

### DISCUSSION THREAD

**Discussion by Bizzik** [Guilouz/Creality-Helper-Script-Wiki#481] [Features Request]
I appreciated when the installer recognized the CR-10 SE as 'K1,' but now it detects it as 'KE.' This detection change has led to the absence of features like Screws Tilt Adjust support. Is it possible to have it detected as 'K1' again?

**Comment by gasbasd** [comment]
i think the best solution is to implement a specific config for cr-10 se, @Guilouz i'm open to give help and i have that printer

**Reply by maxzera** [reply]
Has this ever been implemented?
